README.md

```diff
@@ -248,16 +248,24 @@ func application(_ application: UIApplication, continue userActivity: NSUserActi
 	return [IterableAPI handleUniversalLink:userActivity.webpageURL];
 }
 ```
+
+### Deferred deep linking
+
+If an app user does not have your app installed, deep linking will not work. The user will need to install your app first. After installing your app, the user will be taken to a specific location within your app when they launch it for the first time. See definition of deferred deep linking [here](#https://en.wikipedia.org/wiki/Deferred_deep_linking).
+
+#### Enabling deferred deep linking
+
+Set `IterableConfig.checkForDeferredDeeplink = true` to enable deferred deep linking for IterableSDK.
```

Comment:
- Is "IterableSDK" the name of the SDK? It reads a little awkwardly, but if that's the title, then 👍

Reply:
In cocoapods it is called 'Iterable-iOS-SDK'. In your code the name of the module is 'IterableSDK'.  That is what you use to import our SDK in code. I think we should call it 'IterableSDK'.